src/components/settings-menu-item.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * @file settings-menu-item.js
+ */
+
+import videojs from 'video.js';
+
+const MenuItem = videojs.getComponent('MenuItem');
+const playbackRateMenuButton = videojs.getComponent('PlaybackRateMenuButton');
+const settingsSubMenuTitleEl = Symbol('settingsSubMenuTitleEl');
```

Comment:
Is `Symbol` going to be available via babel in IE8?
I think it's fine if it doesn't work there for new features but if we can support it with *minimal* work, that would be good.